include/zephyr/bluetooth/services/ots.h

```diff
@@ -1029,6 +1056,23 @@ int bt_ots_client_write_object_data(struct bt_ots_client *otc_inst, struct bt_co
 				    const void *buf, size_t len, off_t offset,
 				    enum bt_ots_oacp_write_op_mode mode);
 
+/** @brief Get the checksum of the current selected object.
+ *
+ *  This will trigger an OACP calculate checksum operation for the current object
+ *  with a specified offset and length.
+ *
+ *  The checksum goes to OACP IND and obj_data_checksum() callback.
```

Comment:
Does the `obj_data_checksum()` reference work the callback? Don't you need to do something like `bt_ots_client_cb.obj_data_checksum()` or similar to generate a proper reference to the callback in the doxygen documentation?

Reply:
Modify description to
` *  The checksum goes to OACP IND and obj_checksum_calculated() callback.`